### PR TITLE
Uniform batch scripts

### PR DESCRIPTION
**Pull request by KmaTTi89**
- fix for build.bat and package.bat

### CODE REVIEW

File: package.bat [hunk 1]
```diff
@@ -1,2 +1,5 @@
-cd src\Website
-starpack -p
\ No newline at end of file
+@ECHO OFF
+
+PUSHD %~dp0src\Website
+starpack -p
+POPD
```
Comment (warpech) [inline package.bat]: This repo has two projects: Website and WebsiteProvider. I suppose that you need to add:

```
+PUSHD %~dp0src\WebsiteProvider
 +starpack -p
 +POPD
```

File: build.bat [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+) ELSE (
+    ECHO Error: You don't seem to have Visual Studio 2015 or 2017 installed
+)
+
+:: Try to restore packages. This is only needed when the solution depends on NuGet packages
+PUSHD %~dp0\tools
+WHERE nuget.exe >nul 2>nul
+IF %ERRORLEVEL% EQU 0 nuget.exe restore ..
+IF NOT EXIST "..\packages\" (ECHO Error: Get nuget.exe or build the sln in VS to restore the packages && EXIT /B 1)
+POPD
```
Comment (warpech) [inline build.bat]: Are there any NuGet packages used in this project?

File: package.bat (hunk 1 above)
Comment (KmaTTi89) [inline package.bat]: Fixed

File: build.bat (hunk 2 above)
Comment (KmaTTi89) [inline build.bat]: Fixed

**Pull request merged**